operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -201,6 +201,8 @@ limits_config:
   cardinality_limit: {{ .Stack.Limits.Global.QueryLimits.CardinalityLimit }}
   max_streams_matchers_per_query: 1000
   query_timeout: {{ .Stack.Limits.Global.QueryLimits.QueryTimeout }}
+  volume_enabled: true
+  volume_max_series: 1000
```

Comment:
The default value needs to be replaced by the CRD variable now.

The default values are placed in the sizes.go